unlock-protocol-com/blog/subgraph-v2.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Subgraph upgrade
+subTitle: An upgraded version of the subgraph is now available
+publishedDate: Sep 29 2022
+authorName: Clement Renaud
+description: An upgraded version of the subgraph which indexes onchain data is now available, with simpler data structure and more robust parsing.
+image:
+---
+
+## Storing and retrieving data from chains 
+
+As the number of features grow, keeping track of changes in a contract can become tricky. The transaction data itself only hold changes in balances of native tokens of the sender and receiver. Changes in the state of the contract storage are recorded using events emitted by the transaction. For instance, a ownership change of an NFT will emit a `Transfer` event containing the id of the token, as well as the addresses of the previous and new owners. 
```

Comment:
grow => grows _(a weird one...the subject here is "number" (singular) even though the word before grow ("features") is plural)_

a ownership change => an ownership change

Reply:
I keep forgetting these 3rd person `s` ...